package.json

```diff
@@ -26,8 +26,10 @@
     "clean": "rm -rf _site/ && find src/ \\( -name '*.js' -o -name '*.js.map' \\) -delete",
     "deploy-test": "npm run -s netlify -- deploy -m \"Test deployment\" --alias",
     "netlify": "netlify",
-    "storybook": "start-storybook -p 6006 --no-dll",
-    "build-storybook": "build-storybook --no-dll"
+    "prestorybook": "tsc -p tsconfig.browser.json && wca analyze 'src/**/*.ts' --format json --outFile custom-elements.json",
+    "storybook": "start-storybook",
```

Comment:
This is the process guts and might be part of me not fully groking the project...
1. You need to be sure that all of your TS is built.
2. You need to be sure that a `custom-element.json` files has been created form the TS.
3. Then, and only then, start Storybook.